streams/us_moveonjoy.m3u

```diff
@@ -9,6 +11,8 @@ http://fl3.moveonjoy.com/Aspire/index.m3u8
 http://fl2.moveonjoy.com/BBC_AMERICA/index.m3u8
 #EXTINF:-1 tvg-id="BBCNewsNorthAmerica.uk",BBC News North America
 http://fl2.moveonjoy.com/BBC_WORLD_NEWS/index.m3u8
+#EXTINF:-1 tvg-id="beINSportsUSA.us",beIN Sports (720p)
+http://fl2.moveonjoy.com/BEIN_SPORTS/index.m3u8
```

Comment:
Not quite sure about this one chief. Many beIN channels are DMCAd.

Reply:
eg. https://github.com/iptv-org/database/blob/2682df62bbd03c6d34fc4c7b582045f8a7691754/data/blocklist.csv#L150

Reply:
I was wondering why it isn't marked as so.